easybuild/easyconfigs/n/nodejs/nodejs-12.16.1-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'nodejs'
+version = '12.16.1'  # 12.16.1 was LTS on 2020-05-22
+local_libversion = '72'
+
+homepage = 'http://%(name)s.org'
```

Comment:
just use `nodejs.org` here, so the URL is copy-pasteable (matters less for `source_urls`)